weighted_graph/weight_graph.py

```diff
@@ -0,0 +1,94 @@
+# _*_ coding: utf-8 _*_
+"""Graph Data Structure."""
+
+
+class Graph(object):
+    """Class of the Graph Data Structure."""
+
+    def __init__(self):
+        """Initialize graph."""
+        self.graph = {}
+
+    def nodes(self):
+        """List nodes in graph."""
+        return list(self.graph.keys())
+
+    def edges(self):
```

Comment:
This is just as broken as the simple graph implementation of the edges method. This only returns a list of nodes, not of actual edges